.github/workflows/tests.yml

```
@@ -361,15 +346,13 @@ jobs:
             pytest $DEBUG $COVERAGE $RERUNS --junitxml=../tests/junit/test-results11.xml test_custom_type_field/.
 
       - name: "Kill all servers"
-        uses: ansys/pydpf-actions/kill-dpf-servers@v2.2
-        if: always()
+        uses: ansys/pydpf-actions/kill-dpf-servers@v2.3
 
       - name: "Upload Test Results"
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: ${{ env.PACKAGE_NAME }}_${{ matrix.python-version }}_${{ matrix.os }}_pytest_${{ inputs.ANSYS_VERSION }}
+          name: ${{ env.PACKAGE_NAME }}_${{ matrix.python-version }}_${{ matrix.os }}_pytest_${{ inputs.ANSYS_VERSION }}_${{ inputs.test_any == 'true' && 'any' || '' }}
```

If this condition concatenates string (which I don't know), in some cases there will be a dangling underscore. Is that intended?
Did you also test that it concatenates strings as you most probably want?
```suggestion
          name: ${{ env.PACKAGE_NAME }}_${{ matrix.python-version }}_${{ matrix.os }}_pytest_${{ inputs.ANSYS_VERSION }}${{ inputs.test_any == 'true' && '_any' || '' }}
```

---

@BClappe yes, It does indeed concatenates at the right time. I did miss the trailing underscore though, thanks.